src/Stratis.Bitcoin.Features.Miner/Staking/PosMinting.cs

```diff
@@ -760,6 +747,37 @@ private void CheckStake(Block block, ChainedHeader chainTip)
             return true;
         }
 
+        internal Transaction PrepareCoinStakeTransactions(int currentChainHeight, CoinstakeContext coinstakeContext, long coinstakeOutputValue, int utxosCount, long amountStaked)
+        {
+            this.logger.LogTrace("({0}:{1},{2}:{3},{4}:{5},{6}:{7})", nameof(currentChainHeight), currentChainHeight, nameof(coinstakeOutputValue), coinstakeOutputValue, nameof(utxosCount), utxosCount, nameof(amountStaked), amountStaked);
+
+            // Split stake into SplitFactor utxos if above threshold.
+            bool shouldSplitStake = this.ShouldSplitStake(utxosCount, amountStaked, coinstakeOutputValue, currentChainHeight);
+
+            if (!shouldSplitStake)
+            {
+                coinstakeContext.CoinstakeTx.Outputs[1].Value = coinstakeOutputValue;
+                this.logger.LogTrace("Coinstake output value is {0}.", coinstakeContext.CoinstakeTx.Outputs[1].Value);
+                this.logger.LogTrace("(-):{0}", coinstakeContext.CoinstakeTx);
```

Comment:
this.logger.LogTrace("(-)[NO_SPLIT]:{0}", coinstakeContext.CoinstakeTx);

Reply:
Hey Apro, I have fixed this one in another coming PR that builds on top of this one
https://github.com/stratisproject/StratisBitcoinFullNode/pull/1780

Reply:
ok, that's fine